compose/cli/command.py

```diff
@@ -80,6 +80,7 @@ def get_client(environment, verbose=False, version=None, tls_config=None, host=N
         version=version, tls_config=tls_config, host=host,
         environment=environment, tls_version=get_tls_version(environment)
     )
+    warn_for_swarm_mode(client)
```

Comment:
Sorry, I just meant that a warning on `up` would be enough. Reading it again I realized it wasn't clear.

I think we still need to warn even if there are containers. They might have started the containers before enabling swarm mode. Then if they run `up` again (thinking it will move the containers onto other nodes or something), the warning would be important.